styles/colors-carbon.less

```diff
@@ -98,6 +98,7 @@
 @agate-item-label-color: @agate-accent;
 @agate-item-focus-bg-color: @agate-spotlight-bg-color;
 @agate-item-focus-color: @agate-spotlight-color;
+@agate-item-label-focus-selected-color: @agate-spotlight-color;
```

Comment:
this variable is not declare in colors-base.less. Can you declare it there 
@agate-item-label-focus-selected-color: @agate-spotlight-color;
and remove it from the files of the skins which us the same value?